server/game/cardaction.js

```diff
@@ -20,6 +20,7 @@ class CardAction {
     constructor(game, card, properties) {
         this.game = game;
         this.card = card;
+        this.canPlayFrom = properties.canPlayFrom || 'play';
```

Comment:
'play area' always, until such time as we change it all and have a mapping between 'play' and 'play area'